packages/snaps-controllers/src/interface/SnapInterfaceController.test.tsx

```diff
@@ -545,6 +545,60 @@ describe('SnapInterfaceController', () => {
       expect(state).toStrictEqual({ foo: { baz: null } });
     });
 
+    it('can update an interface and context', async () => {
```

Comment:
Can you add a test to make sure the old context is used when no new context is provided?

Reply:
Sure!